src/viking.js

```diff
@@ -1,11 +1,102 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
```

Comment:
could also use the shorthand `this.health -= damage` to achieve the same purpose!